tests/integrated/test-backtrace/runtest

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -ex
+
+#requires all_tests
+
+e=1
+make || e=$?
```

Comment:
This won't work if using CMake and a non-makefile build generator, so it might be easier to convert this to a python script